torch/csrc/distributed/rpc/init.cpp

```diff
@@ -253,6 +253,16 @@ If the future completes with an error, an exception is thrown.
           &ProcessGroupAgent::sync,
           py::call_guard<py::gil_scoped_release>());
 
+  module.def(
+      "get_current_rpc_agent",
+      &RpcAgent::getDefaultRpcAgent,
+      py::call_guard<py::gil_scoped_release>());
```

Comment:
Should we not release GIL on these two functions to avoid ctx switch as they are very light?

Reply:
Make sense to me, let me remove.